filebeat/docs/fields.asciidoc

```diff
@@ -3233,6 +3233,22 @@ haproxy Module
 
 
 
+*`haproxy.destination_port`*::
++
+--
+type: long
+
+Port of the destination host
+
+--
+
+*`haproxy.destination_ip`*::
++
+--
+IP of the destination host
+
+--
+
```

Comment:
Use `haproxy.destination.port` and `haproxy.destination.ip`.

Reply:
Done in 2c75aa5130a2c48c512bb469d83650ddd316f3c8